internal/api/middleware_test.go

```diff
@@ -185,7 +181,7 @@ func (ts *MiddlewareTestSuite) TestLimitEmailOrPhoneSentHandler() {
 			req.Header.Set("Content-Type", "application/json")
 			w := httptest.NewRecorder()
 
-			for i := 0; i < 5; i++ {
+			for i := 0; i < 30; i++ {
```

Comment:
/nit would be nice to specify where this number comes from (e.g. bursts) or define a constant for it